src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -86,9 +86,6 @@ internal PublicClientApplicationBuilder WithBroker(bool enableBroker)
             return this;
         }
 
-#if !NET_CORE_BUILDTIME // include for other platforms and for runtime
```

Comment:
Why would we expose this ? We can leave it enabled at runtime, but we should hide it at build time.

Reply:
Yes, I think this is what should be done - enable it on netstandard and iOS, hide it at buildtime on the other platforms.